web/index.php

```diff
@@ -166,7 +168,7 @@ function cors() {
 }
 
 // return HTTP 200 for HTTP OPTIONS requests
-$app->map('/:x+', function($x) {
+$app->map('/:x+', 'cors', function($x) {
```

Comment:
Can we rename 'x' to something a bit more descriptive? Perhaps 'routes' or 'optionRoutes'